KarateTraining/src/test/java/abmarticle/abmarticle.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Testing the react herokuapp for creation , modification and removal
+
+  Background:
+    * url baseUrl
+    * def token = authInfo.accessToken
+
+  @CreateNewArticle
+  Scenario: Create a new article
+
+    * def article = read('classpath:abmarticle/abmNewArticleRequest.json')
+
+    Given path 'articles'
+    And request article
+    And header Authorization = token
+    When method post
+    Then status 200
+    Then match response == read('classpath:abmarticle/abmNewArticleResponseSchema.json')
+    And assert response.article['title'] == article['article']['title']
+    And assert response.article['description'] == article['article']['description']
+    And assert response.article['body'] == article['article']['body']
+    And def getActualArticles = call read('abmarticle.feature@getAllArticles')
+    And def addedArticle = karate.jsonPath(getActualArticles.response, "$.articles[?(@.title=='" + article['article'].title + "')]")[0]
```

Comment:
if I'm not wrong, you can get the article by slug. Why don't you set this value on the request?